doc/nrf/libraries/caf/sensor_manager.rst

```diff
@@ -1,13 +1,13 @@
-.. _caf_sensor_sampler:
+.. _caf_sensor_manager:
```

Comment:
Forgot: please keep the old tag, so that the page can be retrieved from older releases.
See this page as reference: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/_sources/ug_matter.rst.txt

Reply:
Fixed